ncl-core/src/main/java/com/paywithmybank/ncl/loader/ClassModuleLoader.java

```diff
@@ -2,25 +2,49 @@
 
 import com.paywithmybank.ncl.model.Source;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class ClassModuleLoader implements ModuleLoader {
 	Class<?> loader;
 	String path;
 	
 	public ClassModuleLoader() {
 		this.loader = this.getClass();
-		this.path = "/";
+		this.path = null;
```

Comment:
This way I can use `null` or blank to indicate that a path wasn't defined and apply legacy mode instead in compliance with this requirement on [this ticket](https://trustly.atlassian.net/browse/DEV-225195):

> Control that the loading will be by file or by legacy mode

Behavior won't change because the `openModule` method was altered to append `/` if `path` is equal to `null`